src/perftest_resources.c

```diff
@@ -80,28 +78,30 @@ static int pp_init_gpu(struct pingpong_context *ctx, size_t _size)
 		return 1;
 	} else if (deviceCount == 1)
 		printf("There is 1 device supporting CUDA\n");
-	else
-		printf("There are %d devices supporting CUDA, picking first...\n", deviceCount);
-
-	int devID = 0;
+	else if (deviceCount > devID)
```

Comment:
here you compare deviceCount with deviceID,  could you explain if this reamingin from this commit when you refer deviceID as deviceCount? 
please explain HL what is the meaning of this testing ?

Reply:
I don't remember refering deviceID as deviceCount. 

The second commit it is just fixing the description to to the --help output for the new parameters and fixes an error with the parser routine where the parser did not expect a second argument after the the --cuda_device

The comparison is required to make sure the user specified an recognized device. If I use a server with 2 GPUs I can pick device 0 or 1 but device 2 would result an error. Nothing fancy.

Reply:
could there be situation where we support more the 2 GPU?  
please add the possibility also of 0 GPU's.

Reply:
0 GPUs (assuming Cuda is available) will result the error:
Error- There are 0 devices supporting CUDA, can not pick device %d

Regarding more than two GPUs- I don't see why not.